### DISCUSSION TITLE
Is there any way to do word diff (ex: --color-words)?

### DISCUSSION THREAD

**Discussion by Jas-SinghFSU** [jesseduffield/lazygit#3415] [Q&A] [answered]
When I use the side by side (or inline) diff tool in lazy git, it shows giant green or red blobs for changes that are simply me putting code inside a function.

In VSCode, the diff shows as me just adding the function name at the top and a curly brace at the bottom but anything in between is simply white because it didn't change (if I'm just wrapping my code in a function).

However, lazygit shows a giant red block as if I deleted all that code and then a giant green blob of all that same code with the function name added to the top.

In git diff I can do --color-words and it makes the diff much more readable, but I prefer the UI of lazygit. Is there any way I can make the lazygit diff perform this way so I can get meaningful diffs?

**Comment by stefanhaller** [comment, marked as answer]
You could try if pressing ctrl-w (to turn "ignore whitespace" mode on) improves it.

**Reply by Jas-SinghFSU** [reply]
Yup, that was it. Thank you so much!
